packages/plugin-electron-deliver-minidumps/deliver-minidumps.js

```diff
@@ -28,15 +33,84 @@ module.exports = (app, net, filestore, NativeClient) => ({
       client._config.maxBreadcrumbs
     )
 
+    const { sendMinidump } = sendMinidumpFactory(net, client)
+    const queue = new MinidumpQueue(filestore)
+    const loop = new MinidumpDeliveryLoop(sendMinidump, client._config.onSend, queue, client._logger)
+
     app.on('ready', () => {
+      isNetworkReady = true
```

Comment:
This might be poor naming on my part - I meant that you could call `net` functions safely. Though yeah, it makes sense to plug it into the network status checker to see if anything can actually be sent.